src/toast/pipeline_tools/sky_signal.py

```diff
@@ -284,12 +328,12 @@ def scan_sky_signal(
 
 @function_timer
 def simulate_sky_signal(
-    args, comm, data, focalplanes, cache_prefix, verbose=False, pixels="pixels", mc=0,
+    args, comm, data, focalplanes, cache_prefix, verbose=False, pixels="pixels", mc=0
 ):
     """ Use PySM to simulate smoothed sky signal.
 
     """
-    if not args.pysm_model:
+    if not args.pysm_model or not args.simulate_sky:
         return None
```

Comment:
As You are modifying this, what about replace return None with return only?

Reply:
Explicit is better than implicit (https://www.python.org/dev/peps/pep-0020) especially when the method usually has a return value. Now that you brought this up,  I'll go add `None` to https://github.com/hpc4cmb/toast/blob/sky_signal_args/src/toast/pipeline_tools/sky_signal.py#L252-L255

Reply:
Conviqt and PySM args already add these: https://github.com/hpc4cmb/toast/blob/sky_signal_args/src/toast/pipeline_tools/sky_signal.py#L136 and https://github.com/hpc4cmb/toast/blob/sky_signal_args/src/toast/pipeline_tools/sky_signal.py#L245 respectively.

Reply:
I agree with the first answer to https://stackoverflow.com/questions/15300550/return-return-none-and-no-return-at-all.
Here we are breaking out of a func so I would use return.